ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/Footer.kt

```diff
@@ -0,0 +1,156 @@
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.revenuecat.purchases.paywalls.PaywallData
+import com.revenuecat.purchases.ui.revenuecatui.R
+import com.revenuecat.purchases.ui.revenuecatui.UIConstant
+import com.revenuecat.purchases.ui.revenuecatui.data.MockViewModel
+import com.revenuecat.purchases.ui.revenuecatui.data.PaywallViewModel
+import com.revenuecat.purchases.ui.revenuecatui.data.TestData
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.ColorsFactory
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.TemplateConfiguration
+import java.net.URL
+
+@Composable
+internal fun Footer(
+    templateConfiguration: TemplateConfiguration,
+    viewModel: PaywallViewModel
+) {
+    Footer(
+        configuration = templateConfiguration.configuration,
+        colors = templateConfiguration.getCurrentColors(),
+        viewModel = viewModel
+    )
+}
+
+@Composable
+internal fun Footer(
+    configuration: PaywallData.Configuration,
+    colors: TemplateConfiguration.Colors,
+    viewModel: PaywallViewModel
+) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(intrinsicSize = IntrinsicSize.Max)
+            .padding(
+                start = UIConstant.defaultHorizontalPadding,
+                end = UIConstant.defaultHorizontalPadding,
+                bottom = UIConstant.defaultButtonVerticalSpacing,
+            ),
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically,
+    ) {
+        val color = colors.text1
+
+        if (configuration.displayRestorePurchases) {
+            Button(
+                text = stringResource(id = R.string.restore_purchases),
+                color = color,
+                action = { viewModel.restorePurchases() }
+            )
+
+            if (configuration.termsOfServiceURL != null || configuration.privacyURL != null) {
+                Separator(color = color)
+            }
+        }
+
+        configuration.termsOfServiceURL?.let {
+            Button(
+                text = stringResource(id = R.string.terms_and_conditions),
+                color = color,
+                action = { viewModel.openURL(it) }
+            )
+
+            if (configuration.privacyURL != null) {
+                Separator(color = color)
+            }
+        }
+
+        configuration.privacyURL?.let {
+            Button(
+                text = stringResource(id = R.string.privacy_policy),
+                color = color,
+                action = { viewModel.openURL(it) }
+            )
+        }
+    }
+}
+
+@Composable
+private fun RowScope.Separator(color: Color) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentSize(Alignment.Center)
+            .weight(0.5f)
```

Comment:
Same as in the other PR, I think you can move these to the `Box` inside and remove the `Column`. A `Column` or `Row` with a single item is "usually" not needed

Reply:
That makes the shape take the full width:

![Screenshot 2023-09-19 at 12 32 55](https://github.com/RevenueCat/purchases-android/assets/685609/f27f5f51-0755-462b-8bcf-92394e503a2b)